src/main/java/com/palantir/docker/compose/execution/AggressiveShutdownStrategy.java

```diff
@@ -27,25 +27,15 @@ public void shutdown(DockerComposeRule rule) throws IOException, InterruptedExce
         List<ContainerName> runningContainers = rule.dockerCompose().ps();
 
         log.info("Shutting down {}", runningContainers.stream().map(ContainerName::semanticName).collect(toList()));
-        if (removeContainersCatchingErrors(rule, runningContainers)) {
-            return;
-        }
-
-        log.debug("First shutdown attempted failed due to btrfs volume error... retrying");
-        if (removeContainersCatchingErrors(rule, runningContainers)) {
-            return;
-        }
-
-        log.warn("Couldn't shut down containers due to btrfs volume error, "
-                + "see https://circleci.com/docs/docker-btrfs-error/ for more info.");
+        removeContainersCatchingErrors(rule, runningContainers);
```

Comment:
Cool.  Are you happy to copy your proposed ShutdownStrategy into your repo for a few builds?